setup.md

```diff
@@ -2,48 +2,84 @@
+### Install Podman on Linux
 
-~~~bash
-docker run hello-world
-~~~
+Podman is available on the official repositories of most Linux distributions. Check the
+[official documentation](https://podman.io/docs/installation#installing-on-linux)
+to find out how to install it on your system.
 
-### Optional: Fetch docker images in advance
+For example, in Ubuntu you can install it by running the following command:
+```bash
+# Ubuntu 20.10 and newer
+sudo apt-get update
+sudo apt-get -y install podman
+```
 
-Once you've got docker up and running, do the following docker image pulls in advance to save waiting during the tutorial:
+### Install Podman on MacOS
 
-~~~bash
-docker pull matthewfeickert/intro-to-docker
-docker pull debian:buster-slim
-docker pull python:2.7-slim
-docker pull python:3.7-slim
-docker pull rootproject/root:6.22.06-conda
-~~~
+Running Podman or Docker on MacOS requires a virtual machine to run the containers.
+In the case of Podman, it provides an installer at https://podman.io/. Download the `.dmg` package for MacOS, extract
+it and execute Podman Desktop.
 
-## Analysis Code
+The first time that Podman Desktop is executed it will require to
+install Podman and a Podman machine to execute the containers. Follow the instructions at the Setup.
+
+
+### Install Podman on Windows
+
+Podman provides instructions to install it on Windows at the [GitHub repository](https://github.com/containers/podman/blob/main/docs/tutorials/podman-for-windows.md).
 
-Later in this tutorial, you will be asked to work with a simple analysis that utilizes the CMS OpenData to search for Higgs to 2 tau leptons.
-The full analysis itself can be found [here](https://github.com/hsf-training/hsf-training-cms-analysis) - and there is a dedicated set of [training lessons](https://hsf-training.github.io/hsf-training-cms-analysis-webpage/index.html) ([videos available](https://www.youtube.com/watch?v=gplMywJAFDI&list=PLKZ9c4ONm-Vk0wnDKaaovoEkOk3PVdL0V)).
-These are then used as input to the [continuous integration training](https://hsf-training.github.io/hsf-training-cicd/) ([videos available](https://www.youtube.com/watch?v=NcVGX8zWzQY&list=PLKZ9c4ONm-VmmTObyNWpz4hB3Hgx8ZWSb)).
 
-Though it is best if you work through these lessons, the key point is that we will assume you have two GitLab repos in your namespace :
+> ## Installing Docker
+>
+> If you still prefer to use Docker (check with the IT department of your institution before using Docker!),
+> follow the official instructions for [Linux](https://docs.docker.com/engine/install/#server), [Mac](https://docs.docker.com/desktop/install/mac-install/), or [Windows](https://docs.docker.com/desktop/install/windows-install/).
+>
+> If you are using Linux, then please also follow these [post installation instructions](https://docs.docker.com/engine/install/linux-postinstall/).
+>
+> Across the tutorial, just replace `podman` by `docker` in the commands and you should be good to go.
+>{: .source}
+{: .callout}
```

Comment:
The way this is formatted makes it look like it's part of the "Install Podman on Windows" section.